h2o-core/src/test/java/water/runner/H2ORunner.java

```diff
@@ -28,6 +27,8 @@
 @Ignore
 public class H2ORunner extends BlockJUnit4ClassRunner {
     private final TestClass testClass;
+    private final HashSet doOnlyTestNames;
```

Comment:
why not add <String> generics?

Reply:
I also have no idea why I did not put it there. Thanks for noticing it. :man_facepalming: 

Fixed in: https://github.com/h2oai/h2o-3/pull/4598/commits/c74cd201174a6cb0766b1a513c4efed37842218f